dataduct/config/config.py

```diff
@@ -75,7 +79,7 @@ def __init__(self, mode=None):
     def __str__(self):
         """String output for the config object
         """
-        return yaml.dump(self._root_config, default_flow_style=False)
+        return yaml.dump(self._root_config, default_flow_style=False, indent=4)
```

Comment:
Shouldn't it be `indent=2`?
Our config files are formatted with 2 spaces, see `.travis.yaml`